### PR TITLE
feat: add monitor

### PR DESCRIPTION
**Pull request by solargatsby**

### CODE REVIEW

**Review by fpChan — APPROVED** [review]

File: offchain-modules/packages/scripts/src/integration.ts [hunk 1]
```diff
@@ -177,6 +203,13 @@ async function startService(
     )}`,
     false,
   );
+  await execShellCmd(
```
Comment (huwenchao) [inline offchain-modules/packages/scripts/src/integration.ts:206]: What's the point of add monitor to integration test?
IMO, it's a background process and have no influence to the test.

File: offchain-modules/packages/x/src/config.ts [hunk 2]
```diff
@@ -163,6 +163,14 @@ export interface collectorConfig {
   gasPriceGweiLimit: number;
 }
 
+export interface monitorConfig {
+  expiredTime: number;
+  expiredCheckInterval: number;
+  discordWebHook: string;
+  scanStep: number;
+  env: string;
+}
```
Comment (huwenchao) [inline offchain-modules/packages/x/src/config.ts:172]: I recommend to use a separate config for monitor.
It's too heavy to include the whole ForceBridgeCore to just use it's config.

Comment (solargatsby) [reply offchain-modules/packages/x/src/config.ts:172]: separate config also have many config item，such as ckb deps group，white list，and multisigLockscript，ownerCellTypescript。It won't too much less then ForceBridgeConfig

File: offchain-modules/packages/scripts/src/integration.ts (hunk 1 above)
Comment (solargatsby) [reply offchain-modules/packages/scripts/src/integration.ts:206]: OK，I will remove it from integration test

**Pull request merged**